contracts/AutomatedFunctionsConsumer.sol

```diff
@@ -44,7 +44,7 @@ contract AutomatedFunctionsConsumer is FunctionsClient, ConfirmedOwner, Automati
   ) FunctionsClient(router) ConfirmedOwner(msg.sender) {
     donId = _donId;
     s_fulfillGasLimit = _fulfillGasLimit;
-    s_lastUpkeepTimeStamp = block.timestamp;
+    s_lastUpkeepTimeStamp = 0;
```

Comment:
@KuphJr 
Is this only so that it works on the testnet? does the testnet ganache not come with its own starting timestamp || 0? this deviates a bit from our docs etc and may be confusing but if unavoidable, is ok.

Reply:
This is because I want to be able to trigger an upkeep immediately after I deploy. I don't want to wait for the full time interval before I am able to call `performUpkeep` (or have the Automation nodes call it)